pkg/appregistry/input_test.go

```diff
@@ -6,11 +6,29 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestDuplicatePackage(t *testing.T) {
+func TestDuplicatePackageResultHasNoRelease(t *testing.T) {
 	parser := inputParser{sourceSpecifier: &registrySpecifier{}}
-	expectedPackages := []string{"Jaeger"}
+	expectedPackages := []*Package{&Package{Name: "Jaeger", Release: ""}}
 
-	actual, err := parser.Parse([]string{"https://quay.io/cnr|community-operator|"}, "Jaeger,Jaeger")
+	actual, err := parser.Parse([]string{"https://quay.io/cnr|community-operator|"}, "Jaeger,Jaeger,Jaeger:0.0.1,Jaeger:0.0.2")
```

Comment:
Can you explain these two test cases? Why is the order important?

Reply:
These tests ensure that existing behavior persists. When a packageList provided to `sanatizePackageList` contains multiple packages with the same name, only the first instance of the package is added. These test together ensure that behavior persists with the change to add a `Package struct` that allows users to specify a `release`.